README.md

```diff
@@ -15,9 +15,13 @@ Install the package with:
 npm i --save @descope/node-sdk
 ```
 
-## Setup
+## Authentication Functions
 
-A Descope `Project ID` is required to initialize the SDK. Find it on the
+### Setup
+
+Before you can use authentication functions listed below, you must initialize a `DescopeClient`.
```

Comment:
I find this text a bit intimidating,  
I would just say something like
 - Make sure you have a Descope `Project ID`, you can find it on the ...
 - Create an sdk with the project ID
 
 ```typescript
const descopeClient = DescopeClient({ projectId: 'my-project-ID' });
```